operator/data/profiles/preview.yaml

```diff
@@ -0,0 +1,6 @@
+# The preview profile contains features that are experimental.
+# This is intended to explore new features coming to Istio.
+# Stability, security, and performance are not guaranteed - use at your own risk.
+apiVersion: operator.istio.io/v1alpha1
+kind: IstioOperator
+spec: {}
```

Comment:
@howardjohn Are there other promoted defaults for 1.5 that should be reverted (and added here instead)?

Reply:
The only ones I know of are WASM which is apparently not (yet) ready to be added here yet. So for now this is empty I suppose